oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,450 @@
+highest level, the platform is composed of numerous services (IDAs
+[#ida]_), micro-frontends (MFEs [#mfe]_) and libraries.  The code for these
+services and libraries is managed in git and hosted on the GitHub
+platform.
+
+Each component part of the Open edX platform requires maintenance to
+ensure that it meets, and continues to meet, the project definition of
+high quality.  That definition will include project standards,
+appropriate documentation, and fitness for use, among other things.
+That definition will evolve as project needs do and we learn what
+works and what doesn't.
+
+Component maintenance is work.  To ensure that that work is done, a
+person or persons must be recognized as responsible for doing it,
+and held accountable for its performance.
+
+The Open edX project, like many projects, calls this role
+"maintainer."  The notion of a project maintainer is typical of
+open-source projects and widely considered a best practice [#maintainer]_.
+
+The maintainer role is a leadership role that encompasses both
+technical and community management elements.  Both dimensions of the role are
+equally important.
+
+All non-deprecated components of the Open edX project must have one
+or more named maintainers.  When no such maintainer(s) exist, this will
+be considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will be open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in any particular community member's production
+environment [#ops]_.
+
+Motivation
+==========
+
+With the change in stewardship from edX to The Center for Reimagining
+Learning, it is essential to have a well defined plan for maintaining
+the platform.
+
+In order to ensure that the project is sustainable for the long term,
+maintenance must be shared by the entire community.  The project must
+be robust enough that it can tolerate firms and contributors leaving
+the ecosystem.
+
+As a project we must acknowledge that maintenance is hard work, but
+work that must be done for the platform to be fit for use.  The best
+way of ensuring this is to have a named person or persons who are
+responsible for getting the work done, a measurable definition of
+"well-maintained," and feedback looks to ensure accountability and
+continuous improvement of our processes.
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the `openedx GitHub organization`_. Components will be "part
```

Comment:
What repositories are you thinking we'd want maintainers for that are outside of the `openedx` organization?

Based on the wording of **Bootstrapping**, I think this OEP would require maintainers for all public repos in the edx organization. Do we want to go down that path?

The community also depends on repos in the `overhangio` organization, namely tutor and its official plugins. I believe @regisb has established his own system of maintainers, though, so I assume  that those repos don't fall in the scope of this OEP.

Reply:
I want to provide for things like hooks and signals, Blockstore, etc.  I realized those have migrated into the openedx org, but there are recent examples of critical components that were not in that org.  I want to make sure that if and when that case arises we have congruent expectations around maintenance.

Reply:
Resolving for future discussion in OEP-57.